projects/configs/mr-spectroscopy.yaml

```
@@ -0,0 +1,35 @@
+#  Copyright (c) 2022 University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+project:
+    name: "mr-spectroscopy"
+    modalities: ["MR"]
+
+tag_operation_files:
+    base:
+        - "base.yaml" #Expected base config file for any project
+        - "mri.yaml"
+        - "ms-pinpoint.yaml"
+        - "mr-spectroscopy.yaml"
+    manufacturer_overrides: ["mri.yaml"]
+
+series_filters:
+    - "localizer"
+    - "localiser"
+    - "scout"
+    - "positioning"
+
+destination:
+    dicom: "dicomweb"
```

Shouldn't be exporting anything at this point
```suggestion
    dicom: "none"
```